Import Error due to invalid directory in package.json

![imageedit_5_5566789286](https://user-images.githubusercontent.com/74496738/155324262-66ea0dbd-c54c-4afd-b37d-8dd4e892b30c.jpg)

The directories of files in package.json is of the format ```lib/cjs/<filename>```
while in the package directory they are located at ```lib/cjs/src/<filename>```
which leads to the error ```Cannot find module 'nodeclub/node_modules/casdoor-js-sdk/lib/cjs/index.js'. Please verify that the package.json has a valid "main" entry```

manually changing the package.json then gives correct import results

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -8,7 +8,6 @@
     "declaration": true,
     "downlevelIteration": true,
     "allowSyntheticDefaultImports": true,
-    "rootDirs": ["./src", "./test"]
   },
-  "include": ["src/**/*", "test/**/*"]
+  "include": ["src/**/*"]
 }

